sesman/startwm.sh

```diff
@@ -3,6 +3,8 @@
 #start the window manager
 wm_start()
 {
+  dd if=/dev/urandom bs=16 count=1 2>/dev/null | od -x -An | tr -d ' ' | sed -e 's/^/add '$DISPLAY' . /' | xauth -q
```

Comment:
xauth has an option to generate the cookie. That might be a cleaner approach.

I believe `startwm.sh` should abort in case of failure. Fallback to insecure operation is not a good idea.